src/binary/non_blocking/raw_binary_reader.rs

```diff
@@ -1204,7 +1209,7 @@ impl<'a, A: AsRef<[u8]>> TxReader<'a, A> {
         let expected_value_length = annotations_and_value_length
             - annotations_length.size_in_bytes()
             - annotations_length.value();
-        self.tx_buffer.total_consumed();
```

Comment:
🗺️ Removing a stray accessor function that had no impact on the method logic (and was optimized out of the assembly anyway).